adapter/internal/discovery/xds/server.go

```diff
@@ -357,7 +357,10 @@ func GenerateEnvoyResoucesForLabel(label string) ([]types.Resource, []types.Reso
 			clusterArray = append(clusterArray, openAPIClustersMap[apiKey]...)
 			routeArray = append(routeArray, openAPIRoutesMap[apiKey]...)
 			endpointArray = append(endpointArray, openAPIEndpointsMap[apiKey]...)
-			apis = append(apis, openAPIEnforcerApisMap[apiKey])
+			enfocerAPI, err := openAPIEnforcerApisMap[apiKey]
+			if err {
```

Comment:
Do we add the API in case of error

Reply:
variable is not an error rather `ok` or `found`